image-archiver.sh

```diff
@@ -33,25 +33,25 @@ if [ ! -d $image_path ]; then
 fi
 
 if [ ! -d $archive_mount ]; then
-	echo $archive_mount "does not exist"
-	exit 1
+	echo $archive_mount "does not exist, trying to mount.."
+	mount $archive_mount || exit 1
```

Comment:
This is probably not a good idea. We want the script to bail if the environment is not set up because something is wrong.